pyeasee/easee.py

```diff
@@ -242,6 +242,7 @@ async def _refresh_token(self):
             res = await self.session.post(
                 f"{self.base}/api/accounts/refresh_token", headers=self.minimal_headers, json=data
             )
+            await raise_for_status(res)
```

Comment:
Good catch!
Suggest that you break out this change to a separate PR since it is not related to the type hint syntax